vpr/src/route/route_timing.cpp

```diff
@@ -159,6 +163,10 @@ static void prune_unused_non_configurable_nets(CBRR& connections_inf);
 static void init_net_delay_from_lookahead(const RouterLookahead& router_lookahead,
                                           ClbNetPinsMatrix<float>& net_delay);
 
+//debug related functions
+void update_router_info_and_check(char type, int net_id);
```

Comment:
and_check part seems vague.  update_router_info_and_check_bp would be better.  Or route_check_bp ().

Reply:
resolved